sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/ValueStateSuite.scala

```diff
@@ -356,7 +357,8 @@ class ValueStateSuite extends StateVariableSuiteBase {
       // ttl value should still exist in state
       ttlValue = nextBatchTestState.getTTLValue()
       assert(ttlValue.isDefined)
-      assert(ttlValue.get === ttlExpirationMs)
+      val ttlExpiration2 = ttlValue.get._2
+      assert(ttlExpiration2 === ttlExpirationMs)
```

Comment:
Can we add some more unit tests in ValueStateSuite/ListStateSuite/StatefulProcessorHandleSuite ?